explanation/anbox-cloud.md

```diff
@@ -13,47 +17,61 @@ The Anbox Cloud Appliance is a self-contained, single-machine deployment variant
 
 The regular Anbox Cloud uses [Juju](https://juju.is/) for deployment and operations. It provides rich features and is ready made for a large scale deployment.
 
+See the following table for a comparison of features for the different variants:
+
+| Feature | Anbox Cloud Appliance | Anbox Cloud |
+|---------|-----------------------|-------------|
+| [Streaming capabilities](https://discourse.ubuntu.com/t/streaming-android-applications/17769) | ✓ | ✓ |
+| [Web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871) | ✓ | ✓ |
+| [Android API version](https://discourse.ubuntu.com/t/provided-images/24185) | 10, 11 | 10, 11 |
+| [Long term security updates](https://discourse.ubuntu.com/t/supported-versions/21046) | ✓ | ✓ |
+| [Community support](https://discourse.ubuntu.com/c/anbox-cloud/) | ✓ | ✓ |
+| [Vendor support](https://anbox-cloud.io/contact-us) | - | ✓ |
+| [Monitoring](https://discourse.ubuntu.com/t/monitor-anbox-cloud/24338) | - | ✓ |
```

Comment:
Is the link correct for what we mean with monitoring here?

Reply:
Yes, we will need to adjust the content behind this but that looks good.